src/Sarif/Core/StackFrame.cs

```diff
@@ -55,10 +55,7 @@ public static StackFrame Create(System.Diagnostics.StackFrame dotNetStackFrame)
 
             if (nativeOffset != -1)
             {
-                stackFrame.Properties = new Dictionary<string, string>
-                {
-                    { "NativeOffset", nativeOffset.ToString(CultureInfo.InvariantCulture) }
-                };
+                stackFrame.SetProperty("NativeOffset", nativeOffset.ToString(CultureInfo.InvariantCulture));
```

Comment:
This could be an integer property if we supported explicitly setting non-string-valued properties.